src/main/java/pro/trevor/tankgame/rule/impl/version3/Tank3.java

```diff
@@ -0,0 +1,62 @@
+package pro.trevor.tankgame.rule.impl.version3;
+
+import org.json.JSONObject;
+import pro.trevor.tankgame.state.board.unit.tank.GenericTank;
+import pro.trevor.tankgame.state.board.unit.tank.IAttributeDecoder;
+import pro.trevor.tankgame.state.board.unit.tank.status.IStatusDecoder;
+
+public class Tank3 extends GenericTank<TankAttribute> {
```

Comment:
You mentioned that naming this Tank3 caused the types to be changed to tank3 can this be named Tank since you replaced the old version?